apple/swift-docc
# [Feature] Add svg and gif support for DocC image(SR-15311)

Feature/SR-15311

## Summary

Swift-DocC currently ignores SVG and GIF files.

This PR add support for recognizing them as images in the same we do for JPEG, JPG, and PNG files.

## Testing

Steps:

1. Download sample project developer.apple.com/documentation/xcode/slothcreator_building_docc_documentation_in_xcode
2. Download a svg file and a gif file and drag them to the Package's Sources/SlothCreator/SlothCreator.docc/Resources folder
3. Replace the content in SlothCreator.tutorial
```swift
@Image(source: slothcreator-intro.png, alt: "An illustration of ... tutorials.")
```
to
```swift
@Image(source: xx.svg, alt: "An illustration of ... tutorials.")
```
```swift
@Image(source: xx.gif, alt: "An illustration of ... tutorials.")
```
4. Build documentation to verify.

## Checklist

Make sure you check off the following items. If they cannot be completed, provide a reason.
